docs/millennium-os/manual/chapters/40_gcodes.md

```diff
@@ -229,7 +229,7 @@ If the post knows what _type_ of workpiece probe should be executed, it can call
 G6500 [Wnn]
 ```
 
-Guided bore probe, prompts the user for approximate diameter, overtravel, approximate center position and probe depth.
+The guided Bore probe prompts the user for approximate diameter, overtravel, approximate center position, and probe depth.
```

Comment:
Style seems inconsistent - can we potentially capitalise the name like so:

> - **Guided Bore Probe** prompts the user 

bold would help pick out the command your looking for